pilot/pkg/model/config.go

```diff
@@ -1113,12 +1114,16 @@ func (store *istioConfigStore) ServiceRoleBindings(namespace string) []Config {
 	return bindings
 }
 
-func (store *istioConfigStore) RbacConfig(name, namespace string) *Config {
-	rbacConfigs, err := store.List(RbacConfig.Type, namespace)
+func (store *istioConfigStore) RbacConfig(name string) *Config {
+	rbacConfigs, err := store.List(RbacConfig.Type, "")
 	if err != nil {
 		log.Errorf("failed to get rbacConfig: %v", err)
 		return nil
 	}
+	if len(rbacConfigs) > 1 {
```

Comment:
Would it be better to ignore all, except the one with name == DefaultRbacConfigName ? You should not have one with name other than DefaultRbacConfigName  anyway, as validation should catch it. Anyway, ignore those should be safe enough.

Reply:
Done, only do a log for this.